lib/src/fonts.dart

```diff
@@ -6,96 +6,48 @@ class GYWFont {
   /// The name of the font
   final String name;
 
-  /// The internal prefix of the font
-  final String prefix;
-
-  /// The size (in points) of a character
-  final int size;
-
-  /// The width (in pixels) of a character
-  final int width;
-
-  /// The height (in pixels) of a character
-  final int height;
-
-  /// Whether the fontweight is bold (w700) or normal (w400)
-  final bool bold;
+  /// The font filename on the GYW device
+  final String filename;
 
   /// Creates a font.
   const GYWFont({
     required this.name,
-    required this.prefix,
-    required this.size,
-    required this.width,
-    required this.height,
-    required this.bold,
+    required this.filename,
   });
 
   @override
   String toString() {
-    return "Font '$name' ($prefix)";
+    return "Font '$name' ($filename)";
   }
 
   @override
   bool operator ==(Object other) {
     if (other is GYWFont) {
-      return prefix == other.prefix;
+      return filename == other.filename;
     } else {
       return false;
     }
   }
 
   @override
-  int get hashCode => Object.hash(prefix, size);
+  int get hashCode => Object.hash(name, filename);
 }
 
 /// The [GYWFont] fonts supported by default on aRdent smart glasses.
 enum GYWFonts {
-  /// For small texts with a low significance.
-  small(
-    GYWFont(
-      name: "Small",
-      prefix: "a10",
-      size: 18,
-      width: 10,
-      height: 25,
-      bold: false,
-    ),
-  ),
-
-  /// For common texts with a medium significance.
-  medium(
-    GYWFont(
-      name: "Medium",
-      prefix: "b14",
-      size: 24,
-      width: 14,
-      height: 33,
-      bold: true,
-    ),
-  ),
-
-  /// For common texts with a medium significance.
-  large(
+  /// Roboto Mono Normal
+  robotoMonoNormal(
     GYWFont(
```

Comment:
Shouldn't we keep `bold` and `italic` parameters ? and have something like `pathSvg` that returns the font to use in Flutter preview ?

Reply:
> Sure. Is it just because you want to offer more information to the user just in case?

I want the user to easily 'convert' his text in a preview

> `pathTtf` you meant probably. Or should we just name it `path`?

You can name it  `path`. But in reality, i'm thinking that when we use the font in a `TextStyle` object, we never use the ttf value. So I'm not sure if it is really relevant to have this. The goal is to but the value in the `TextStyle` object, what do you suggest ?

Reply:
Perfect, font family seems perfect. With that, bold and italic will make more sense as we may have a font family that is linked to several `GYWFont`